src/_data/styles/stageflix.json

```
@@ -0,0 +1,5 @@
+{
+  "title": "Stageflix",
+  "author": "Suckcoder",
+  "stylesheet": "https://codepen.io/suckcoder/pen/KKVbObL.css",
```

```suggestion
  "stylesheet": "https://codepen.io/suckcoder/pen/KKVbObL.css"
```

Remove the comma and then it should work!